typescript/src/commands/General/Chat Bot Info/help.ts

```diff
@@ -54,7 +55,11 @@ export class UserCommand extends SkyraCommand {
 
 		// Handle case for a single command
 		const command = await args.pickResult('commandName');
-		if (command.success) return message.send(await this.buildCommandHelp(message, args.t, command.value, context.commandPrefix));
+		const prefix =
+			(context.prefix instanceof RegExp && !context.commandPrefix.endsWith(' ')) || UserOrMemberMentionRegex.test(context.commandPrefix)
+				? `${context.commandPrefix} `
+				: context.commandPrefix;
```

Comment:
Can you move all of this logic to a separate method? e.g. `getCommandPrefix`

Reply:
done